spec/controllers/admin/enterprise_fees_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: false
+
+require 'spec_helper'
+require 'open_food_network/order_cycle_permissions'
+
+RSpec.describe Admin::EnterpriseFeesController do
+  before { @request.env['HTTP_REFERER'] = 'http://test.com/' }
+
+  before do
+    allow(controller).to receive_messages spree_current_user: super_admin
```

Comment:
I think in general it's better to test with a non-admin user (eg a user that is a manager of the enterprise) to ensure they have the correct permission. But I wouldn't bother changing it now.

I can see that the below feature spec uses an enterprise user, so all good 👍